employees/admin.py

```diff
@@ -7,7 +7,10 @@
 
 
 class TaskActivities(admin.ModelAdmin):
-    list_display = ("name",)
+    list_display = ("name", "is_active")
+
+    def has_delete_permission(self, request, obj=TaskActivityType) -> bool:
```

Comment:
Unnecessary `obj=TaskActivityType`. If you wanted to add type annotations you need to change it -> `obj: TaskActivityType`

Reply:
changed